extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RangerSparkExtensionSuite.scala

```diff
@@ -316,6 +316,41 @@ abstract class RangerSparkExtensionSuite extends KyuubiFunSuite with SparkSessio
       doAs("admin", sql(s"DROP DATABASE IF EXISTS $db3"))
     }
   }
+
+  test("KYUUBI #2596 Support to control the viewing permissions of " +
```

Comment:
we have already tested databases, and tables?

Can we add a test for columns in an independent test?

Reply:
okay